frontend/src/test/setup.ts

```diff
@@ -0,0 +1,8 @@
+import "@testing-library/jest-dom";
```

Comment:
This adds nice DOM element matchers to Vitest that make tests more readable and expressive.
- `toBeInTheDocument()`
- `toHaveClass()`
- `toHaveAttribute()`
- `toBeVisible()`
- `toBeChecked()`